clowdapp.yaml

```diff
@@ -130,7 +130,7 @@ parameters:
   value: "True"
 - description: The interval at which garbage collection should run (in seconds)
   name: GARBAGE_COLLECTION_INTERVAL
-  value: "86400"
+  value: '86400'
 - description: Number of days after which data is considered to be outdated
   name: DAYS_UNTIL_STALE
-  value: "45"
+  value: '45'
```

Comment:
@kgaikwad Are the single quotes necessary? I can see double quotes line # 130

Reply:
Yes, not necessary. Reverted